app/src/main/java/com/example/adrian/workshop1/Profil.java

```diff
@@ -0,0 +1,149 @@
+package com.example.adrian.workshop1;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.v7.app.AppCompatActivity;
+import android.text.Html;
+import android.text.SpannableString;
+import android.text.Spanned;
+import android.text.style.UnderlineSpan;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.example.adrian.workshop1.model.GitHub;
+import com.example.adrian.workshop1.model.ProfileData;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+/**
+ * Created by Adrian on 3/28/2017.
+ */
+
+public class Profil extends AppCompatActivity implements View.OnClickListener {
+
+    private TextView Location;
+    private TextView Email;
+    private TextView Description;
+    private TextView Name;
+    private TextView Employee;
+    private TextView Created;
+    private TextView Updated;
+    private TextView PublicRepos;
+    private TextView PrivateRepos;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        // Get the view from profil.xml
+        setContentView(R.layout.profil);
+
+        findViewById(R.id.blog_button).setOnClickListener(this);
+
+        getProfile();
+    }
+
+    private void getProfile() {
+        SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(this);
+        Call<ProfileData> callable = GitHub.Service.Get().getProfile(pref.getString("login", null));
+
+        callable.enqueue(new Callback<ProfileData>() {
+            @Override
+            public void onResponse(Call<ProfileData> call, Response<ProfileData> response) {
+                if(response.isSuccessful()) {
+                    updateUI(response.body());
+                } else {
+                    Toast.makeText(Profil.this, "Couldn't fetch profile", Toast.LENGTH_LONG).show();
+                }
+            }
+
+            @Override
+            public void onFailure(Call<ProfileData> call, Throwable t) {
+                t.printStackTrace();
+                Toast.makeText(Profil.this, "Internet problem", Toast.LENGTH_LONG).show();
+            }
+        });
+    }
+
+    private void updateUI(ProfileData data) {
+        Location = (TextView) findViewById(R.id.location);
```

Comment:
E de preferat sa faci findViewById in onCreate. Tinand cont ca la tine updateUI se apeleaza o singura data e ok, dar in cazul in care o vom face mai des, nu vrei sa-ti obtii view-urile de fiecare data.